styles/bsky/catppuccin.user.css

```diff
@@ -98,10 +98,8 @@
     [style*="color: rgb(11, 15, 20)"] {
       color: @text !important;
 
-      /* fix contrast issues */
-      & when not (@lookup = latte) {
-        color: @crust !important;
-      }
+      // fix contrast issues on latte
+      color: if(@lookup = latte, @text, @crust) !important;
```

Comment:
You only need the latter `color` line here, the first one is redundant.

Reply:
Of course the diff doesn't match what I'm looking at -_- but I mean the line with `color: @text !important;`, right before this "fix contrast issues on latte" comment.

Reply:
oh, oops. i meant to remove that lmao